src/routes/state-management.lazy.tsx

```diff
@@ -0,0 +1,86 @@
+
+
+import { createLazyFileRoute } from '@tanstack/react-router';
```

Comment:
Hi @OsamaAbdellateef , can you move this content to /components -> new folder state-management -> state-management.tsx and than call this in this state-management.lazy.tsx? 

So that we keep the structure of the reference apps the same? :)

Reply:
I read ur comment on ref app ticket in Jira and you can have a look on the structure now , 
didn't add much stuff for codhike because we said we will figure it out in pair prog if you have anything i can add to the structure please let me know  @sarahgm

Reply:
Yes I only meant this, for me this comment is resolved now :) Only want to make sure to have codehike inside of it. Good job!